src/counters.c

```diff
@@ -234,9 +234,15 @@ static void StatsInitCtx(void)
             SCLogDebug("Stats module has been disabled");
             SCReturn;
         }
-        const char *interval = ConfNodeLookupChildValue(stats, "interval");
-        if (interval != NULL)
-            stats_tts = (uint32_t) atoi(interval);
+        
+        intmax_t interval;
+        if (ConfGetChildValueInt(stats, "interval", &interval) == 0) {
+            stats_enabled = FALSE;
+            SCLogDebug("Stats module has been disabled, invalid interval");
```

Comment:
Agree on that, should it be a fatal error message (SC_ERR_FATAL) and not some kind of fallback to the default value, since maybe other systems / scripts / etc, could be dependant on the correct stats output interval.